src/SharedFixerConfig.php

```diff
@@ -0,0 +1,40 @@
+<?php
```

Comment:
Question:
should we make `SharedFixerConfig ` or `WhitespacesFixerConfig` ?
(mimic the potential chaneg to `...AwareInterface`)